src/classes/ADDR_HouseholdAcc_TEST.cls

```diff
@@ -1851,9 +1851,16 @@ public with sharing class ADDR_HouseholdAcc_TEST {
         Test.stopTest();
         
         // verify the Address objects split street.
-        List<Address__c> listAddr = [select Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c from Address__c 
-        where MailingStreet__c = 'new street'];
+        List<Address__c> addresses = [SELECT Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c from Address__c];
```

Comment:
Curious about this. Why do we need to create a new list of address? Can we loop through the addresses and update the mailing street field?

Reply:
Originally there was a filter on this query: "MailingStreet__c = 'new street'" however we have to remove that or Shield encryption will fail. Therefore, not knowing exactly what could be coming back from the select, I wanted to ensure that we were creating a list that exactly matches what the original query was intending which was to only contain addresses whose Mailing Street was 'new street' I am basically 'guaranteeing' that the data I'm working with is exactly the data I need.
We could probably loop through and update but I felt this more exactly duplicated the intent of using the filtered query.